### ISSUE TITLE
Ubuntu 18.04 still on 7-rc.3

### ISSUE THREAD

**Issue by JustinGrote** [PowerShell/PowerShell-Docker#391]
# Repro

```
docker run mcr.microsoft.com/powershell:lts-ubuntu-18.04
```

# Expected
to run 7.0.0

# Actual

```
Status: Downloaded newer image for mcr.microsoft.com/powershell:lts-ubuntu-18.04
PowerShell 7.0.0-rc.3
Copyright (c) Microsoft Corporation. All rights reserved.

https://aka.ms/powershell
Type 'help' to get help.
```

# Orignial 

![image](https://user-images.githubusercontent.com/15258962/76706860-d8238700-66a7-11ea-9836-1ef50c5c3281.png)
EDIT: Same with the 1809 nanoserver images

**Comment by TravisEz13** [comment]
Please include a repro.  Are you using our containers?

**Comment by TravisEz13** [comment]
I already have a redeployment started.  I'll hopefully be able to get it done soon.

**Comment by TravisEz13** [comment]
Believe this is resolved.

**Comment by TravisEz13** [comment]
Looking back... Our logs show we published an image containing 7.0.0 into the ACR which MCR replicates.  Because I had already started another deployment, I cannot precisely say where the problem happened.   I've opened this issue to help diagnose this in the future: https://github.com/PowerShell/PowerShell-Docker/issues/393
